app/components/MachineCarousel.tsx

```diff
@@ -83,7 +86,7 @@ export default function MachineCarousel() {
     nextArrow: (
       <NextArrow
         className={""}
-        onClick={(event) => {
+        onClick={() => {
```

Comment:
That's pre-commit hook with ESLint at work. I wonder why it did this for me, but not for you Brett, when you committed those changes. Do you have a separate config file for ESLint or some setting in VScode that takes precedence? Or does this happen only to me?

Reply:
Not sure exactly. Here is what I have in my VSCode settings:
<img width="463" alt="Screenshot 2024-02-17 at 12 13 23 PM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/76603041/805d6744-f2cf-408a-a8f0-2e5946a1032b">

Reply:
Hm, this is really odd. I have the same settings. Oh well, it doesn't seem like it's a major issue, so I'm not going to investigate it further if you don't mind :)